k8s/upgrades/0.8.0-0.8.1/cstor_volume_upgrade.sh

```diff
@@ -64,8 +68,8 @@ sc_res_ver=`kubectl get sc $sc_name -n $sc_ns -o jsonpath="{.metadata.resourceVe
 # c-dep: pvc-cec8e86d-0bcc-11e8-be1c-000c298ff5fc-target        # 
 #################################################################
 
-c_dep=$(echo $pv-target)
-c_svc=$(echo $pv)
+c_dep=$(kubectl get deploy -n $ns -l openebs.io/persistent-volume=$pv,openebs.io/target=cstor-target -o jsonpath="{.items[*].metadata.name}")
+c_svc=$(kubectl get svc -n $ns -l openebs.io/persistent-volume=$pv,openebs.io/target-service=cstor-target-svc -o jsonpath="{.items[*].metadata.name}")
 c_vol=$(echo $pv)
```

Comment:
I don't think this is in use.. right?

Reply:
we are using it @vishnuitta

Reply:
https://github.com/openebs/openebs/blob/d3be244a79887f92b6a3df9283c6e68b488692d6/k8s/upgrades/0.8.0-0.8.1/cstor_volume_upgrade.sh#L153